certora/specs/StakeManager.spec

```diff
@@ -27,30 +27,32 @@ function getAccountBalance(address addr) returns uint256 {
 
 function isMigrationfunction(method f) returns bool {
   return
-          f.selector == sig:migrateFrom(address,bool,StakeManager.Account).selector ||
-          f.selector == sig:migrateTo(bool).selector;
+          f.selector == sig:migrateTo(bool).selector ||
+          f.selector == sig:transferNonPending().selector;
 }
 
 /*  assume that migration is zero, causing the verification to take into account only
  cases where it is zero. specifically no externall call to the migration contract */
-function simplification() {
+function simplification(env e) {
   require currentContract.migration == 0;
+  require currentContract.oldManager == 0;
+  require e.msg.sender != 0;
```

Comment:
This one here is necessary to account for cases that **theoretically** let old managers reset or decrease a new manager's `currentEpoch`.

Having these requirements causes the prover to find no non-reverting cases for such functions